pulsar-client-api/src/main/java/org/apache/pulsar/client/api/PulsarClientException.java

```diff
@@ -910,6 +910,20 @@ public TransactionConflictException(String msg) {
         }
     }
 
+    /**
+     * Thrown when transaction meta store handler not exists.
+     */
+    public static class MetaStoreHandlerNotExistsException extends PulsarClientException {
```

Comment:
It's a confusing name, the `MetaStoreHandlerNotExistsException` sound like the client will connect to the metadata store, but the metadata store does not exist.

Reply:
This exception is caused by `txnID.getMostSigBits()` > the actual number of TCs.
1. Change `MetaStoreHandlerNotExistsException` to `InvalidMetaStoreHandlerException` or `InvalidTxnIdException`
2. Because in a normal production environment, the situation where `mostSiqBits` is greater than the actual number of TCs will not occur. Consider removing `MetaStoreHandlerNotExistsException`